### PR TITLE
feat: data faker

### LINKED ISSUE #15: Add data faker
Look into [faker.js](https://fakerjs.dev/guide/usage.html) and see if it works with Bun.

### PR DESCRIPTION
**Pull request by aruay99**
### Description
closes #15 
Added a fakerjs that creates 10 fake user data when running:

`bun db:seed`

---

### PR Checklist (Please do not remove)

- [x] Read the [CONTRIBUTING](
  https://github.com/agnyz/elysia-realworld-example-app/blob/main/CONTRIBUTING.md) guide
- [ ] Title this PR according to the `type(scope): description` or `type: description` format
- [ ] Provide description sufficient to understand the changes introduced in this PR, and, if necessary, some screenshots
- [ ] Reference an issue or discussion where the feature or changes have been previously discussed
- [ ] Add a failing test that passes with the changes introduced in this PR, or explain why it's not feasible
- [ ] Add documentation for the feature or changes introduced in this PR to the docs; you can run them with `bun docs`

### CODE REVIEW

**Comment by yamcodes** [comment]
@aruay99, thank you so much for the contribution. 

1. As a convention, I've edited your title from `feat: added data faker` to `feat: data faker`.
  The [the AngularJS Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) linked in our [`CONTRIBUTING.md`](https://github.com/agnyz/elysia-realworld-example-app/blob/main/CONTRIBUTING.md) states:
> ### [Subject](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#subject)
> The subject contains succinct description of the change:
>
> * **use the imperative, present tense: "change" not "changed" nor "changes"**
> * don't capitalize first letter
> * no dot (.) at the end

2. Kindly make sure to handle all items on the checklist in your next PR. This helps us ensure code quality and convention and removes friction.

**Comment by yamcodes** [comment]
I'm testing this PR locally, ideally seed should be idempotent, which means that if it's ran on every boot of the container we shouldn't create new users. 

A potential fix is to truncate the affected table before inserting, which will ensure in this case there is exactly 10 users after every seed.

In that case, we can remove the [`onConflictDoNothing()` statement](https://orm.drizzle.team/docs/insert#on-conflict-do-nothing) here which will become uneccesary.

File: db/seed.ts [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+    username: faker.internet.userName(),
+    password: faker.internet.password(),
+    bio: faker.lorem.text(),
+    image: faker.image.imageUrl(),
+  };
+  console.log('Upserting user:', data);
+
+  await db.insert(users).values(data);
+  console.log('User upserted');
+}
```
Comment (yamcodes) [inline db/seed.ts:23]: Great usage of faker.js

**Review by yamcodes — APPROVED** [review]
LGTM

File: db/seed.ts [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
+console.log('Truncating the user database');
+await db.delete(users);
+console.log('The database is empty: ', await db.select().from(users));
 
+for (let i = 0; i < 10; i++) {
+  const data = {
+    id: faker.datatype.number(),
+    email: faker.internet.email(),
+    username: faker.internet.userName(),
+    password: faker.internet.password(),
```
Comment (yamcodes) [inline db/seed.ts:15]: The password must be hashed. See:
https://github.com/agnyz/elysia-realworld-example-app/blob/6753ffd788e5013f76f514c683acf12822e7d4bb/src/users/users.service.ts#L46

**Review by yamcodes — CHANGES_REQUESTED** [review]
Thanks @aruay99! Note we must hash the passwords that make their way to the DB.

File: db/seed.ts (hunk 2 above)
Comment (aruay99) [reply db/seed.ts]: Thanks! I added hashing

Comment (yamcodes) [reply db/seed.ts]: Well done!

**Review by yamcodes — APPROVED** [review]

**Pull request merged**